s2/point.go

```diff
@@ -160,12 +160,12 @@ func regularPointsForFrame(frame matrix3x3, radius s1.Angle, numVertices int) []
 	z := math.Cos(radius.Radians())
 	r := math.Sin(radius.Radians())
 	radianStep := 2 * math.Pi / float64(numVertices)
-	var vertices []Point
+	var vertices = make([]Point, numVertices)
 
 	for i := 0; i < numVertices; i++ {
```

Comment:
Since we're now accessing vertices[i], this would become more idomatic by iterating over the range instead

for i := range vertices {
...
}